src/procedures/ModifyPercentageExemptions.ts

```diff
@@ -9,9 +9,20 @@ import {
 } from '../types';
 import { PolymathError } from '../PolymathError';
 
+/**
+ * Procedure responsible for modifying any exception related to the Porcentage Transfer Manager module
+ */
 export class ModifyPercentageExemptions extends Procedure<ModifyPercentageExemptionsProcedureArgs> {
   public type = ProcedureType.ModifyPercentageExemptions;
 
+  /**
+   * - Modify whitelist data or allow/disallow the PTM module to mint tokens
```

Comment:
It isn't minting tokens. `allowPrimaryIssuance` is an exempt for issuance operations for this restriction. 
The best use case is when totalSupply is 0. The first issuance will transfer 100% of the tokens, so if you set this module with any percentage, it would fail UNLESS you set `allowPrimaryIssuance` to true.

Also, remove `-` from the beginning.

Reply:
Also, you should refrain from referring to modules or the PTM. Instead talk about percentage restrictions